### PR TITLE
Let MMI brains commit suicide.

### LINKED ISSUE #12845: Ghostly MMI's
**Issue Description**:
Re-Enter your MMI after you suicide.
Basically you can suicide as a brain inside a MMI and just re-enter it. It just turns invisible and doesn't get removed.

**What did you expect to happen**: 
a dead brain inside a mmi

**What happened instead**:
i was able to re-enter the MMI as a ghost

**Why is this bad/What are the consequences**:
very meta gameplay

**Steps to reproduce the problem**:
1. Die
2. Get brain into MMI
3. suicide
4. Press "Re-Enter Corpse"

**When did the problem start happening**:
24.12.2019
17:00

**Extra information**:
Vox can't love.

### PR DESCRIPTION
**Pull request by FunnyMan3595**
## What Does This PR Do
At some point in the past, MMI brains could commit suicide, as evidenced by #12845.  This has been re-added, without that bug.  So, uh, fixes #12845, I guess?

## Why It's Good For The Game
Being in an MMI should not prevent players from becoming an observer.

## Testing
Spawned as robiticist.
Huffed my air tank.
Spawned a new humanoid.
Inserted roboticist brain in MMI.
Took direct control of brain mob.
Committed suicide.
Switched back to the newbie, dumped the brain, put it back.
Took direct control of brain mob again.
Committed suicide again.

## Changelog
:cl:
add: Brains in MMIs can now commit suicide.
/:cl:

### CODE REVIEW

File: code/game/verbs/suicide.dm [hunk 1]
```diff
@@ -158,3 +158,7 @@
 		affected.add_autopsy_data(byitem ? "Suicide by [byitem]" : "Suicide", dmgamt)
 
 	updatehealth()
+
+/mob/living/brain/do_suicide()
+	to_chat(viewers(src), "<span class='danger'>[src] is thinking about thinking about thinking about thinking about.... It looks like [p_theyre()] trying to commit suicide.</span>")
```
Comment (DGamerL) [inline code/game/verbs/suicide.dm:163]:
```suggestion
	to_chat(viewers(src), "<span class='danger'>[src] is thinking about thinking about thinking about thinking about.... It looks like [p_theyre()] trying to commit suicide!</span>")
```

Comment (Henri215) [reply code/game/verbs/suicide.dm:163]: And should probably use 3 periods instead of 4.

Comment (FunnyMan3595) [reply code/game/verbs/suicide.dm:163]: Four is correct, actually.  The first 3 are an ellipsis, and the last on is the period at the end of the sentence.  Three is correct mid-sentence, but the end of a sentence should have three plus a period, question mark, or exclamation point.

File: code/game/verbs/suicide.dm [hunk 2]
```diff
@@ -158,3 +158,7 @@
 		affected.add_autopsy_data(byitem ? "Suicide by [byitem]" : "Suicide", dmgamt)
 
 	updatehealth()
+
+/mob/living/brain/do_suicide()
+	to_chat(viewers(src), "<span class='danger'>[src] is thinking about thinking about thinking about thinking about.... It looks like [p_theyre()] trying to commit suicide!</span>")
```
Comment (lewcc) [inline code/game/verbs/suicide.dm:163]:
```suggestion
	visible_message("<span class='danger'>[src] is thinking about thinking about thinking about thinking about.... It looks like [p_theyre()] trying to commit suicide!</span>")
```

**Review by DGamerL — APPROVED** [review]

**Review by lewcc — APPROVED** [review]

**Pull request merged**